operator/Makefile

```diff
@@ -77,19 +77,19 @@ deploy/deploy-kiali-operator.sh
 ## operator-delete: Remove the Kiali operator resources from the cluster along with Kiali itself
 operator-delete: purge-kiali
 	@echo Remove Operator
-	${OC} delete --ignore-not-found=true all,sa,deployments,clusterroles,clusterrolebindings,customresourcedefinitions --selector=app=kiali-operator -n ${OPERATOR_NAMESPACE}
```

Comment:
put back the "--ignore-not-found=true" option

Reply:
yes.